runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -243,4 +223,37 @@ default int infoBoxSize()
 	{
 		return 35;
 	}
+
+	@ConfigItem(
+		keyName = "overlayFont",
+		name = "Overlay Font",
+		description = "Configures what font type is used for in-game overlays such as player name, ground items, etc.",
+		position = 31
+	)
+	default Font overlayFont()
+	{
+		return FontManager.getRunescapeSmallFont();
```

Comment:
Shouldnt this use new API?

Reply:
Im out of the loop. What new API do you mean?